django/db/models/deletion.py

```diff
@@ -118,8 +118,8 @@ def add_field_update(self, field, value, objs):
 
     def can_fast_delete(self, objs, from_field=None):
         """
-        Determine if the objects in the given queryset-like can be
-        fast-deleted. This can be done if there are no cascades, no
+        Determine if the objects in the given queryset-like or a single object
```

Comment:
Since `objs` isn't used for anything but to get `model`, could you just accept `model` as a kwarg instead, and when it's set ignore `objs`?

Reply:
I could, but this would move the extraction of the model from the queryset-alike to all the calling points; hence I opted for this way.